kernel/td_kernel_dmvw.py

```diff
@@ -168,7 +168,7 @@ def calculate_maps(self):
             self.importance_weight_concentration_map[low_index_x:high_index_x, low_index_y:high_index_y] += omega * self.measurement_concentrations[i]
 
         # alpha
-        self.confidence_map = 1 - np.exp(-self.importance_weight_map / (self.sigma_omega * self.sigma_omega))
+        self.confidence_map = 1 - np.exp(-(self.importance_weight_map)**2 / (self.sigma_omega * self.sigma_omega))
```

Comment:
As pointed out in #1, the non-squared form is preferred using the algorithm with the wind component.